Incremental.c

```diff
@@ -1725,8 +1735,7 @@ int IncrementalRemove(char *devname, char *id_path, int verbose)
 	if (mdfd < 0) {
```

Comment:
is_fd_valid

Reply:
will be reworked in IncrementalRemove PR, I started with, not here.